src/corredor/helpers/compose.ts

```diff
@@ -1075,15 +1076,35 @@ export default class ComposeHelper {
   }
 
   /**
-   * Sets permissions on messaging resources
+   * Creates new allow access permission instance
+   * 
+   * @param role The role that will have access privileges
+   * @param resource The resource that we want to allow for
+   * @param operation The operation that the role can perform
+   */
+  allow (role: Role, resource: Record|Module, operation: string): AllowAccess {
```

Comment:
why just record & module??

Reply:
For the sakes of this POC; in case, I'd have to scratch it